docker-java-transport-httpclient5/src/main/java/com/github/dockerjava/httpclient5/ApacheDockerHttpClientImpl.java

```diff
@@ -66,27 +66,36 @@ protected ApacheDockerHttpClientImpl(
                 host = HttpHost.create(dockerHost);
         }
 
+        PoolingHttpClientConnectionManager connectionPoolMgr = new PoolingHttpClientConnectionManager(
```

Comment:
please keep the variable naming consistent with the rest of the library

Reply:
Please be specific, I cannot guess easily what you want. Tell me what name you like to see and I'll change it.

Reply:
`connectionManager` would be fine, thanks

Reply:
Done!